contrib/utilities/checkdoxygen.py

```diff
@@ -1,30 +1,63 @@
 #!/usr/bin/python
 
-# run this script on all headers of deal.II to check that
-# all doxygen groups have matching @{ @} pairs.
+# run this script on all headers of deal.II to check various doxygen related problems
+
 # for example:
-# find include -name "*h" -print | xargs -n 1 ../tests/scripts/checkdoxygen.py
+# find doc examples include -name "*.h" -o -name "*.dox" -print | xargs -n 1 contrib/utilities/checkdoxygen.py
```

Comment:
I think this should be `find doc examples include -name "*.h" -print -o -name "*.dox" -print`: otherwise we only print the `.dox` files.